api_docs/rest-error-handling.md

```diff
@@ -2,20 +2,40 @@
 
 Zulip's API will always return a JSON format response.
 The HTTP status code indicates whether the request was successful
-(200 = success, 40x = user error, 50x = server error).  Every response
-will contain at least two keys: `msg` (a human-readable error message)
-and `result`, which will be either `error` or `success` (this is
-redundant with the HTTP status code, but is convenient when printing
-responses while debugging).
-
-For some common errors, Zulip provides a `code` attribute.  Where
-present, clients should check `code`, rather than `msg`, when looking
-for specific error conditions, since the `msg` strings are
+(200 = success, 4xx = user error, 5xx = server error).
+
+Every success or error response will contain at least two keys:
+
+- `msg`: an internationalized, human-readable error message string.
+
+- `result`: either `"error"` or `"success"`, which is redundant with the
+  HTTP status code, but is convenient when print debugging.
+
+As of Zulip 5.0 (feature level 76), all error responses also contain a
+`code` key with a machine-readable error string. The default value for
+`code` is `"BAD_REQUEST"`, which is returned for a number of errors.
```

Comment:
Probably this should be explained and formatted as a mandatory key like the previous two, and the pre-76 discussion should be marked as a "Changes" entry.

Reply:
Done in updated version.